agasc/agasc.py

```diff
@@ -39,9 +44,18 @@
     To disable the magnitude / bad star updates from the AGASC supplement, see
     the ``set_supplement_enabled`` context manager / decorator.
 
-    The default ``agasc_file`` is ``<AGASC_DIR>/miniagasc.h5``, where
-    ``<AGASC_DIR>`` is either the ``AGASC_DIR`` environment variable if defined
-    or ``$SKA/data/agasc``.
+    The default for ``agasc_file`` is (in order of precedence):
+    - ``${AGASC_HDF5_FILE}`` if that environment variable is defined.
+    - ``${SKA}/data/agasc/miniagasc_<version>.h5`` where ``<version>`` is the latest
+      version found in the ``${SKA}/data/agasc`` directory. This is the "full" AGASC.
```

Comment:
Nit-picking, the full agasc is the full agasc (agasc<version>.h5).  The quotes around "full" don't really help increase understanding here.  At this point we might put the block from the mini agasc code here 
```
The mini AGASC file is a subset of the full AGASC file that contains only the stars
satisfying ``stars["MAG_ACA"] - 3.0 * stars["MAG_ACA_ERR"] / 100.0 < 11.5``.

The proseco AGASC file is a variation of the mini AGASC file that contains only the
columns needed for proseco. It also adds back the near-neighbor stars (which are close
to a candidate guide or acq star) that got cut by the magnitude filter.
```
depending on where this is going to end up in the docs.  If we are giving users a variety of ways to pick the file it seems like including more information about the files is consistent with the strategy.